src/index.html

```diff
@@ -29,5 +32,6 @@ <h3 class="add-score">Add your score</h3>
       </div>
     </div>
```

Comment:
- [ ] @Lembani Great work, kindly adjust your add your score section, so that the layout matches the design given.


Design  Given       |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/168417711-1435f0f3-654c-4737-b6a7-1c9d79bb9fe1.png) |  ![image](https://user-images.githubusercontent.com/61408860/168417717-8cde2bf7-8e9b-4c6c-9d86-d1b4fe855312.png)

Reply:
Hello @danmainah, Thanks for the feedback.
Can you please check again? 
Here is how it looks from my end: 
<img width="1436" alt="Screenshot 2022-05-14 at 10 06 03 AM" src="https://user-images.githubusercontent.com/30483991/168417912-51dc290e-f668-4f23-8b19-eb07e8127cd8.png">

The above screenshot results from the latest build in the dist folder.

Reply:
I have checked it out , kindly submit for review the project is working

Reply:
Thanks, I have submitted it.